arches_for_science/test_settings.py

```diff
@@ -3,7 +3,6 @@
 # Depends on /arches being on the python path, as arches.tests is not importable
 from tests.test_settings import *
 
-from arches_for_science.settings import INSTALLED_APPS, ROOT_URLCONF, APP_NAME
```

Comment:
This was added recently just for the sake of the project unit tests, so with luck that's all we'll need to get them passing again 🤞